src/server/seeds/development/04_favorites.js

```diff
@@ -0,0 +1,37 @@
+exports.seed = function (knex) {
+  return knex('favorites')
+    .del()
+    .then(function () {
+      return knex('favorites').insert([
+        {
+          user_id: 1,
+          product_id: 1,
+        },
```

Comment:
**created-at** should be in favorites.

Reply:
I think Bikram we dont need created_at here. Because created_at column is using default value

Reply:
@zmzade yes its not nullable, but it have the **default** timestamp value. Thanks for review 👍